netlify.toml

```diff
@@ -4,3 +4,8 @@ command = "npm run build:preview"
 
 [context.production]
 command = "npm run build:production"
+
+[[redirects]]
+from = "https://docs.opentelemetry.io/*"
```

Comment:
The [`from` field is meant to be a path][from], not a URL. Have you read somewhere that it's possible to redirect subdomains in this way?

[from]: https://docs.netlify.com/routing/redirects/#syntax-for-the-netlify-configuration-file

Reply:
I stand corrected, apparently we _can_ redirect subdomains in this way.